azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/model/WebAppWindowsRuntime.java

```diff
@@ -133,7 +134,11 @@ public static WebAppWindowsRuntime fromContainerAndJavaVersion(final String cont
     }
 
     @Nullable
-    public static WebAppWindowsRuntime fromContainerAndJavaVersionUserText(final String containerUserText, final String javaVersionUserText) {
+    public static WebAppWindowsRuntime fromContainerAndJavaVersionUserText(final String containerUserText, String javaVersionUserText) {
+        if (StringUtils.isBlank(javaVersionUserText)) {
```

Comment:
Not sure whether we should add this, because if user just want to deploy to an existing app, it may be a bit confused to show this message.